build-tools/automation/azure-pipelines.yaml

```diff
@@ -44,12 +44,22 @@ variables:
   value: $[or(eq(variables['XA.RunAllTests'], true), eq(variables['IsMonoBranch'], true))]
 - name: DotNetNUnitCategories
   value: '& TestCategory != DotNetIgnore & TestCategory != HybridAOT & TestCategory != ProfiledAOT & TestCategory != MkBundle & TestCategory != MonoSymbolicate & TestCategory != PackagesConfig & TestCategory != StaticProject & TestCategory != Debugger & TestCategory != SystemApplication'
-- ${{ if and(eq(variables['Build.DefinitionName'], 'Xamarin.Android'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+- name: TeamName
+  value: XamarinAndroid
```

Comment:
I think we can cut this, it was moved to https://github.com/xamarin/xamarin-android/blob/main/build-tools/automation/yaml-templates/variables.yaml#L34